### ISSUE TITLE
Flutter app keeps stopping on initialization on Android using 0.5.0

### ISSUE THREAD

**Issue by ebjerkelund** [alann-maulana/flutter_beacon#89]
Building and delivering on Android emulator using VS Code as debugging environment.
When deploying in a debug session on 0.5.0, app will not launch and says that it keeps stopping.
All works fine when using 0.3.0 and no other changes to my flutter app.
Have not tried 0.4.0 yet.

**Comment by alann-maulana** [comment]
Use it at real device instead of emulator

**Comment by ebjerkelund** [comment]
Thanks. It works when I run it on my device.

Is this a problem with the emulator? Or does it need to be accomodated for somehow in code?

Bluetooth support is only a part of the overall functionality of my app. Debugging and iterative development is more of a hassle when you need to use a real device. For now I can use the older version for development as it basically has feature parity for my purposes. Can switch to the latest version for deployment testing and delivery.

**Comment by alann-maulana** [comment]
Yeah, it's already come in my mind too. I'll take a look on a few days.
